sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch

```diff
@@ -0,0 +1,977 @@
+commit 96b0b476522c3f93a47ff8f479ec08105ba6a2a5
```

Comment:
Please add at the top the original URI for each patch, for example `http://sigrok.org/gitweb/?p=sigrok-firmware-fx2lafw.git;a=commitdiff_plain;h=96b0b476522c3f93a47ff8f479ec08105ba6a2a5` or whatever you used to obtain it.

Reply:
Appending them together is still fine yeah?

Reply:
also if you could point me to any documentation around this convention or something if it exists I'de much appreciate it

Reply:
We have some guidelines in the manual, please read the section on [clean patches](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/#clean-patch-howto).  We need to know where the patch comes from, and a link to upstream is best in this case.